privacy-protections/amp/index.html

```diff
@@ -0,0 +1,64 @@
+      </li>
+      <li>
+        <p><a href="https://www.google.fr/amp/s/www.brookings.edu/research/reforming-global-fossil-fuel-subsidies-how-the-united-states-can-restart-international-cooperation/%3famp">Non Standard TLD (Google Domain)</a></p>
+        <p class="expected">Expected: https://www.brookings.edu/research/reforming-global-fossil-fuel-subsidies-how-the-united-states-can-restart-international-cooperation</p>
+      </li>
+      <li>
+        <p><a href="https://www-wpxi-com.cdn.ampproject.org/v/s/www.wpxi.com/news/top-stories/pihl-bans-armstrong-student-section-hockey-games-after-vulgar-chants-directed-female-goalie/G2RP5FZA3ZDYRLFSWGDGQH2MY4/?amp_js_v=a6&amp_gsa=1&outputType=amp&usqp=mq331AQKKAFQArABIIACAw%3D%3D&fbclid=IwAR1TYTsjFEPy42VACpBR0n2jP-6whynOxwVqxpDMTWIRgW2vFlj573xVkEI#aoh=16358554203777&referrer=https%3A%2F%2Fwww.google.com&amp_tf=From%20%251%24s&ampshare=https%3A%2F%2Fwww.wpxi.com%2Fnews%2Ftop-stories%2Fpihl-bans-armstrong-student-section-hockey-games-after-vulgar-chants-directed-female-goalie%2FG2RP5FZA3ZDYRLFSWGDGQH2MY4%2F">ampproject.org with parameters</a></p>
+        <p class="expected">Expected: https://www.wpxi.com/news/top-stories/pihl-bans-armstrong-student-section-hockey-games-after-vulgar-chants-directed-female-goalie/G2RP5FZA3ZDYRLFSWGDGQH2MY4/?amp_js_v=a6&_gsa=1&outputType=amp&usqp=mq331AQKKAFQArABIIACAw%3D%3D&fbclid=IwAR1TYTsjFEPy42VACpBR0n2jP-6whynOxwVqxpDMTWIRgW2vFlj573xVkEI#aoh=16358554203777&referrer=https%3A%2F%2Fwww.google.com&_tf=From%20%251%24s&ampshare=https%3A%2F%2Fwww.wpxi.com%2Fnews%2Ftop-stories%2Fpihl-bans-armstrong-student-section-hockey-games-after-vulgar-chants-directed-female-goalie%2FG2RP5FZA3ZDYRLFSWGDGQH2MY4%2F</p>
+      </li>
+    </ul>
+
+    <p>First Party Cloaked AMP links</p>
+    <ul>
+      <li>
+        <p><a href="https://www.bbc.com/news/world-asia-china-19432800.amp">.amp link</a></p>
+        <p class="expected">Expected: https://www.bbc.com/news/world-asia-china</p>
+      </li>
+      <li>
+        <p><a href="https://amp.theguardian.com/environment/2021/nov/05/british-firm-to-unveil-technology-for-zero-carbon-emission-flights-at-cop26">amp. link</a></p>
+        <p class="expected">Expected: https://www.theguardian.com/environment/2021/nov/05/british-firm-to-unveil-technology-for-zero-carbon-emission-flights-at-cop26</p>
+      </li>
+      <li>
+        <p><a href="https://www.euronews.com/green/amp/2021/11/05/warm-water-how-climate-change-is-affecting-the-baltic-sea">/amp link</a></p>
+        <p class="expected">Expected: https://www.euronews.com/green/2021/11/05/warm-water-how-climate-change-is-affecting-the-baltic-sea</p>
+      </li>
+      <li>
+        <p><a href="https://www.independent.co.uk/news/world/americas/us-politics/covid-vaccine-mandate-kids-san-francisco-b1951340.html?amp">?amp link</a></p>
+        <p class="expected">Expected: https://www.independent.co.uk/news/world/americas/us-politics/covid-vaccine-mandate-kids-san-francisco</p>
```

Comment:
ditto

Reply:
This was a good catch actually. I think independent in this case doesn't have the canonical element loaded when DOMContentLoaded fires. I have an idea to fix this.